### DISCUSSION TITLE
Backup configuration without ReadWriteMany Persistence Volume access mode

### DISCUSSION THREAD

**Discussion by allevimi** [Azure/api-management-self-hosted-gateway#304] [Ideas]
We are working with GKE where persistence access mode ReadWriteMany is not default supported and using ReadWriteOnce with pvc doesn't permit scale on different nodes; we have forked helm chart in order to support a StatefulSet with volumeClaimTemplates, so we have a different PVC for each pod.
It could be a good idea to PR our chart and enable sts support ? Otherwise, you have a better solution in order to achieve our requirements?

**Comment by tomkerkhove** [comment]
ReadWriteMany permissions is required to persist the local configuration so that is required.

This does not mean you need a stateful set though, a deployment is better suites
